tvb_framework/tvb/config/logger/dev_logger_config.conf

```diff
@@ -119,10 +125,16 @@ formatter=simpleFormatter
 # Generate a new file every midnight and keep logs for 1 day
 args=('sqlalchemy.log', 'midnight', 1, 1)
 
+
+[handler_elasticSearchHandler]
+class=tvb.config.logger.elasticsearch_handler.ElasticQueueHandler
+level=DEBUG
```

Comment:
For TRACE logs from `quaelname= tvb_user_actions` to appear in this handler, you should use here also `level=NOTSET`